tuxemon/core/components/event/actions/play_map_animation.py

```diff
@@ -62,7 +60,7 @@ def start(self):
         # "position" can be either a (x, y) tile coordinate or "player"
         animation_name = self.parameters.animation_name
         duration = self.parameters.duration
-        directory = os.path.join(paths.BASEDIR, prepare.DATADIR, "animations/tileset")
+        directory = prepare.fetch("animations", "tileset")
```

Comment:
this looks clean.  i don't think i like that it is `prepare.fetch`, so maybe we can find a better name, but i do think this format would work great.

Reply:
Can be renamed later, too many big changes at once like you said.